src/vivarium_public_health/treatment/intervention.py

```diff
@@ -6,21 +6,45 @@
 
 
 class Intervention(Exposure):
-    """A component for modeling access to an intervention. The modeling and implementation of
-    this component is similar to that of a risk factor where the risk is a loack or not having
-    access to the intervention. Interventions should be configured with in the format of
+    """A model for an intervention defined by coverage (access to intervention).
+
+    This component models access to an intervention as a dichotomous exposure where
+    simulants are either covered or uncovered by the intervention. The modeling
+    and implementation of this component is similar to that of a risk factor where
+    the risk is a lack of or not having access to the intervention.
+
+    For example,
+
+    #. vaccination coverage where simulants are either vaccinated (covered) or
+       unvaccinated (uncovered).
+    #. treatment access where simulants either have access to treatment (covered)
+       or do not have access (uncovered).
+
+    This component can source data either from builder.data or from parameters
+    supplied in the configuration. If data is derived from the configuration, it
+    must be an integer or float expressing the desired coverage level or a
+    covariate name that is intended to be used as a proxy. For example, for an
+    intervention named "intervention", the configuration could look like this:
+
+    .. code-block:: yaml
+
+       configuration:
+           intervention:
```

Comment:
If interventions should be named `intervention.intervention_name`, you should use that convention in this example